SECURITY.md

```
@@ -19,6 +21,16 @@ This section outlines the versions of our project that are currently supported w
 | 1.1.x   | ❌         |
 | 1.0.x   | ❌         |
 
+
+### Key:
```

Markdownlint (0.35.0)

Punctuation: ':'
Trailing punctuation in heading

(MD026, no-trailing-punctuation)